src/detect-filestore.c

```diff
@@ -323,6 +325,17 @@ static int DetectFilestoreSetup (DetectEngineCtx *de_ctx, Signature *s, const ch
 {
     SCEnter();
 
+    bool warn_not_configured = false;
```

Comment:
this will lead to a check per rule with the filestore keyword, right? Making it static will make it a one time check?

Need to consider rule reloads. The output will still be disabled, but we may want to warn at least once per ruleset.

Reply:
I intended for this to be `static` -- will fix and also investigate the rule reload mechanism.

Reply:
Will work this in.